<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Olx Apartment Finder</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Form{
background-color: #1e81b0;
}</string>
  </property>
  <widget class="QListWidget" name="districtsList">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>191</width>
     <height>431</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <property name="flow">
    <enum>QListView::TopToBottom</enum>
   </property>
   <property name="isWrapping" stdset="0">
    <bool>false</bool>
   </property>
   <property name="resizeMode">
    <enum>QListView::Fixed</enum>
   </property>
   <property name="layoutMode">
    <enum>QListView::SinglePass</enum>
   </property>
   <property name="currentRow">
    <number>-1</number>
   </property>
   <item>
    <property name="text">
     <string>All</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bieńczyce</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bieżanów-Prokocim</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bronowice</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Czyżyny</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dębniki</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Grzegórzki</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Krowodrza</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Łagiewniki-Borek Fałęcki</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mistrzejowice</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nowa Huta</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Podgórze</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Podgórze Duchackie</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Prądnik Biały</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Prądnik Czerwony</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Stare Miasto</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Swoszowice</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Wzgórza Krzesławickie</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Zwierzyniec</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="textDistrictsLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Select districts</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="mapBtn">
   <property name="geometry">
    <rect>
     <x>539</x>
     <y>470</y>
     <width>241</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Map of Krakow districts</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="kindLayout">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>190</y>
     <width>271</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>149</width>
      <height>88</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="apartmentRbtn">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>apartment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="roomRbtn">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>room</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="searchLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>121</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Search: </string>
   </property>
  </widget>
  <widget class="QLabel" name="descriptionLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>451</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Dishonest owners only give the price
of the rent, and the rest of the costs
only after entering the ad. Such as
administrative rent, utilities, etc.
            This is the solution</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>340</y>
     <width>401</width>
     <height>116</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="priceLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="minPriceLabel">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Max price:   </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="minPriceInput">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="cursorPosition">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="maxPriceLabel">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Max price:   </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="maxPriceInput">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="cursorPosition">
       <number>0</number>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>530</y>
     <width>741</width>
     <height>46</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="foorLayout">
    <item>
     <widget class="QPushButton" name="searchBtn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>SEARCH</string>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="informationBtn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>INFORMATION</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exitBtn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>EXIT</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
